app/views/services/index.html.erb

```diff
@@ -2,45 +2,35 @@
 
 <% breadcrumb :services %>
 
-<div class="page-title">
-  <h1>Local Links Manager</h1>
-</div>
+<%= render "govuk_publishing_components/components/heading", {
+  text: "Services (#{@services.count})",
+  margin_bottom: 5,
+} %>
 
```

Comment:
Now that 'Local Links Manager' has been included into the header, it appears that all page headings are formatted as `h2`. We should probably change them all to `h1`.
```
<%= render "govuk_publishing_components/components/heading", {
  text: "Services (#{@services.count})",
  heading_level: 1,
  font_size: 'l',
  margin_bottom: 5,
} %>
```

Reply:
Done!